pandas/tests/extension/base/casting.py

```diff
@@ -31,9 +31,7 @@ def test_astype_object_frame(self, all_data):
         assert isinstance(result._mgr.arrays[0], np.ndarray)
         assert result._mgr.arrays[0].dtype == np.dtype(object)
 
-        # earlier numpy raises TypeError on e.g. np.dtype(np.int64) == "Int64"
```

Comment:
Might be good to still keep this comment since we cannot remove the numpy version check here?